pandas/core/generic.py

```diff
@@ -2207,6 +2207,10 @@ def to_json(
 
                 Describing the data, where data component is like ``orient='records'``.
 
+                Note that ``orient='table'`` contains a 'pandas_version' field under
+                'schema'. This denotes the version of `pandas` where the schema was last
```

Comment:
`where the schema was last revised` &#8594; `used in the latest revision of the schema`